distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -457,54 +461,6 @@ type Decimal
     ^ : Number -> Number
     ^ self that = @Builtin_Method "Decimal.^"
 
-    ## Checks if this is greater than that.
```

Comment:
Unfortunately, I had to revert this once again, because it caused a lot of performance regressions in vector operations with primitive values. Explanation in https://github.com/enso-org/enso/pull/5998#issuecomment-1509947907. We can deal with this later, though.

Reply:
I see.